### ISSUE TITLE
Regularization

### ISSUE THREAD

**Issue by lfb94** [thtrieu/darkflow#775]
Hello, 

I'm very new to tensorflow.... I think this is already implemented: https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolov2/train.py#L44

Still I don't understand very well... How could I modify the regularization term importance (weight decay)? Do I need to add the coefficient myself on the final loss function?

Thank you

**Comment by rauar** [comment]
These are the values going into the loss function as penalties: https://github.com/thtrieu/darkflow/blob/b2aee0000cd2a956b9f1de6dbfef94d53158b7d8/darkflow/net/yolov2/train.py#L20-L23

They are defined in the cfg file.
